test/index.spec.ts

```diff
@@ -14,7 +14,7 @@ declare global {
 const testPagePath = path.resolve(__dirname, 'index.html');
 
 const SAMPLE_EXPORTED_KEY =
-  '{"alg":"A256GCM","ext":true,"k":"leW0IR00ACQp3SoWuITXQComCte7lwKLR9ztPlGkFeM","key_ops":["encrypt","decrypt"],"kty":"oct"}';
+  '{"key":{"alg":"A256GCM","ext":true,"k":"leW0IR00ACQp3SoWuITXQComCte7lwKLR9ztPlGkFeM","key_ops":["encrypt","decrypt"],"kty":"oct"}}';
```

Comment:
The key is left unchanged here, we just wrapped it in the new object shape and stringified it again.

This also means that we are testing a scenario where a user has a `JsonWebKey` without any metadata about how it was derived - which we can expect if the user has a key exported with an old version of this library